### DISCUSSION TITLE
Button in Grid not working in view

### DISCUSSION THREAD

**Discussion by qasims** [generic-ui/generic-ui#31] [Q&A]
Is there any way to trigger angular function as below?

{
      header: '',
      field: (item) => item,
      width: 150,
      cellEditing: { enabled: false },
      view: (value: any) => {
        return `
      <button class="btn btn-sm btn-sm btn-danger" type="button" (click)="openConfirmationDialog(${value});">
     Edit
    </button>`;
      }
    }

**Comment by generic-ui** [comment]
You can use mechanism suggested in this thread https://github.com/generic-ui/generic-ui/issues/11#issue-681861327

**Reply by generic-ui** [reply]
We have introduced new mechanism of defining columns in the version 0.15.1. Please see documentation https://generic-ui.com/guide/column-template and an example https://generic-ui.com/examples/agile-users
